tests/pyupgrade_test.py

```diff
@@ -1448,6 +1448,30 @@ def test_fix_native_literals(s, expected):
     assert _fix_py3_plus(s) == expected
 
 
+@pytest.mark.parametrize(
+    ('s', 'expected'),
+    (
+        ('"asd".encode("utf-8")', 'b"asd"'),
```

Comment:
curious what does this does for:

```python
'a' 'b'.encode('UTF-8')

sys.stdout.buffer.write(
    'a'
    'b'.encode('UTF-8')
)
```

Reply:
Looks like my change causes mixing of byte & nonbyte literals. I could not handle this case at all or I could modify the predecessor to be a byte string if possible?

Reply:
maybe let's not convert to bytestrings for now then, and just remove the encoding name -- I think that makes your change simpler / safer anyway 👍

Reply:
suree

Reply:
sure* `!sux mbp`